### PR TITLE
[RPS-474] Cleaning up pipeline resources

### PR DESCRIPTION
**Pull request by wojtek-oledzki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,21 @@ Currently we are supporting two platforms
 
 Run `make help` to get a list of available commands.
 
-For more details on how to use this repo, please read below docs
+In most cases you will run something like this:
+
+```
+# authenticate your terminal to access `nhsd-profile-name` for 1h
+$(make aws-sudo PROFILE=nhsd-profile-name TOKEN=123789)
```
Comment (wojtek-oledzki) [inline README.md]: indicate that "TOKEN" is a AWS MFA token

File: ansible/vars/dev/environment.yml [hunk 2]
```diff
@@ -2,34 +2,19 @@
 
 # global shortcuts
 env: &ENV dev
-config_bucket: "config.{{ env }}.ps.nhsd.io"
+config_bucket: "config.{{ env }}.nhsd.io"
 public_fqdn: &FQDN dev.nhsd.io
 
 aws:
   tags:
     Project: Publication Service
```
Comment (wojtek-oledzki) [inline ansible/vars/dev/environment.yml]: lets change to Digital Website

File: ansible/vars/dev/environment.yml [hunk 3]
```diff
@@ -2,34 +2,19 @@
...
     - *FQDN
 
-hippo_authoring:
-  mysql:
-    host: hippo-authoring.mysql.db.int
-    root_user: root
-    user: hippocms
-    dbname: hippocms
-
 aws_ondemand:
```
Comment (wojtek-oledzki) [inline ansible/vars/dev/environment.yml]: rename to ondemand here and in ondemand role `defaults/main.yml`

File: ansible/vars/mgt/environment.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+
+# global shortcuts
+env: &ENV mgt
+config_bucket: "config.{{ env }}.nhsd.io"
+public_fqdn: &FQDN mgt.nhsd.io
+
+aws:
+  tags:
+    Project: Publication Service
```
Comment (wojtek-oledzki) [inline ansible/vars/mgt/environment.yml]: Digital Website

File: ansible/vars/mgt/environment.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    Environment: *ENV
+
+aws_cd:
+  public_fqdn: "{{ env }}.nhsd.io"
+  pipelines:
+    build:
+      enabled: yes
+      vpc_id: vpc-65b45401
```
Comment (wojtek-oledzki) [inline ansible/vars/mgt/environment.yml]: comment out with descriptive comment explaining when to use

File: docs/aws/cd-pipelines.md [hunk 6]
```diff
@@ -1,5 +1,8 @@
 # Build and Maintain Continuous Deployment Pipelines
 
+
+
```
Comment (wojtek-oledzki) [inline docs/aws/cd-pipelines.md]: more comment

**Review by ngta1 — CHANGES_REQUESTED** [review]
Looks good.  Just need to squash to 1 commit.

**Comment by nhs-digital-website-ps-pipeline** [comment]
@ngta1 squashed

**Review by ngta1 — APPROVED** [review]

**Pull request merged**